packages/perseus/src/widgets/expression/score-expression.ts

```diff
@@ -28,13 +28,12 @@ import type {
  * - For each answer:
  *   ~ Try to validate the user's input against the answer. The answer is
  *     expected to parse.
- *   ~ If the user's input validates (the validator judges it "correct"), we've
- *     matched and can stop considering answers.
```

Comment:
Did you mean to delete this line?

Reply:
Nope, definitely was unintended. I accidentally modified this comment before realizing it was related to khanAnswerType instead of the SSS validators. Returning it now! Thanks for pointing it out!!